stable/airflow/UPGRADE.md

```diff
@@ -1,5 +1,24 @@
 # Upgrading Steps
 
+## `v7.7.X` → `v7.8.0`
+
+> __WARNING:__ 
+>
+> If you install many pip packages with: `airflow.extraPipPackages`, `web.extraPipPackages`, or `dags.installRequirements`
+> 
+> Ensure you set `scheduler.livenessProbe.initialDelaySeconds` to longer than the install time
+>
+
+__The following IMPROVEMENTS have been made:__
+* Upgraded to Airflow: `1.10.12`
+* The scheduler now has a liveness probe which will force the pod to restart if it becomes unhealthy for more than some threshold of time (default: 150sec)
```

Comment:
can you tell that this feature is enabled by default, and the key to disable it.

Reply:
done